frame/staking-rewards/src/weights.rs

```diff
@@ -1,3 +1,11 @@
-pub trait WeightInfo {}
+use frame_support::dispatch::Weight;
 
-impl WeightInfo for () {}
+pub trait WeightInfo {
+	fn create_reward_pool() -> Weight;
+}
+
+impl WeightInfo for () {
+	fn create_reward_pool() -> Weight {
+		todo!()
```

Comment:
not sure, may be put 1000 into pallet, than this. likely CI will not pass now as there are no bench.